numpy/core/tests/test_function_base.py

```diff
@@ -113,6 +113,19 @@ def test_basic(self):
         assert_array_equal(y, [-100, -10, -1])
         assert_array_equal(y.imag, 0)
 
+        y = geomspace(0.3, 20.3, num=1)
```

Comment:
Would be nice to explain what is being tested here.
```suggestion
    def test_exact_limits(self):
        y = geomspace(0.3, 20.3, num=1)
```

Could consider testing a negative limit too.

Reply:
good idea, will do